README.rst

```diff
@@ -89,6 +79,35 @@ in place:
 
 .. installation-end-marker
 
+Contributing
```

Comment:
For consistency: we usually have (1) license before contrib and (2) most have "Contribution" though personally I prefer "Contributing"

Reply:
I will do (1) but not (2)